components/email/src/Email.svelte

```diff
@@ -1916,44 +1956,55 @@
                     </div>
                   {/if}
                   <div class="from-participants">
-                    <div
-                      class="participants-name"
-                      class:condensed={showSecondFromParticipant(
-                        activeThread.messages,
-                        activeThread.participants,
-                      )}
-                    >
-                      <span class="from-sub-section">
-                        {!activeThread.messages.length
-                          ? isDraft
-                            ? "Draft"
-                            : "Deleted Email"
-                          : showFromParticipants(
-                              activeThread.messages,
-                              activeThread.participants,
-                            )}
```

Comment:
- If `isDeleted` is true, display as Deleted Email
- Else
** Render each participants string returned from `getParticipants`, separated with `,`
** "**Draft**" string will be painted red